admin-manual/installation-setup/integrations/duracloud-setup.rst

```diff
@@ -1,226 +1,107 @@
-                      Digital objects
-            /Project2
-                 /Text files
-                      Digital objects
-                 /Word files
-                      Digital objects
-
-In this example, the archivist will have more flexibility when deciding which
-directory becomes a transfer in Archivematica. Either Project1 and Project2
-could be transfers, as in the examples above, or the subdirectories within
-could become their own transfers (Photographs, Videos, etc).
-
-**Sync an existing directory**
-
-1. If there are existing directories that you wish to sync but do not wish to    reorganize into a directory structure deep enough to work with Archivematica,    you can instead use the Sync Tool's prefix option:
-
-  * Ensure your sync tool is stopped in the Status tab. Then click on the
-     configuration tab.
-
-  * Under "Other options," create a prefix for your sync folder to create a
-      directory structure at least two levels deep. The prefix must end in a
-      slash (/).
-
-2. Note that the prefix will replace the directory name of the sync folder in
-   DuraCloud.
-
-Example 1:
-
-.. code:: bash
-
-   /syncFolder
-      Digital objects
-
-If the sync directory selected in the sync tool is ``syncFolder``, you could add
-a prefix such as: ``transfers/Project1/``. Archivematica would then recognize
-``transfers`` as a transfer source, and ``Project1`` would be available as a
-transfer. That transfer would contain all of the digital objects in ``syncFolder``.
-
-3. It is also possible to use the prefix option with a sync folder which has
-   subfolders.
-
-Example 2:
-
-.. code:: bash
-
-   /syncFolder
-        /Photographs
-             Digital objects
-        /Videos
-             Digital objects
-
-If the sync directory selected in the sync tool is ``syncFolder``, you could add
-a prefix such as: ``transfers/Project1``. In this example, ``transfers`` will
-still be interpreted by Archivematica as a transfer source, but either
-``Project1``, ``Photographs`` or ``Video`` could be chosen in the dashboard as
-the transfer.
-
-Step 5: Configure transfer sources
-----------------------------------
-
-Assuming that you have configured transfer-source spaces in DuraCloud,
-return to the Archivematica Storage Serice at this point to configure transfer
-sources locations.
-
-Navigate to the transfer-source Storage Service space and create locations for
-each top level directory inside the directory or directories you have synced.
-
-.. tip::
-
-   If you consistently use the same named top-level directory when syncing
-   (e.g. "transfers") you will only need to configure the transfer source
-   once.
-
-.. tip::
-   If you browse for a path and do not see the expected directory listed, this
-   may be caused by a UI bug. You can type the path in instead.
+#. In the Storage Service's Spaces tab, click **Create new space** and enter the
+   following information:
+
+   * **Access protocol**: DuraCloud
```

Comment:
Should this section just link to the Storage Service docs instead? https://www.archivematica.org/en/docs/storage-service-0.15/administrators/#duracloud

Reply:
(or should this information be moved and updated there?, is the better question)

Reply:
Good point @ablwr, thanks! I just went with what was already on the page but pointing out to the SS docs to reduce the number of places we need to keep updated is better.